models/stg_google_ads__ad_history.sql

```diff
@@ -29,8 +29,10 @@ final as (
         type as ad_type,
         status as ad_status,
         display_url,
-        final_urls as source_final_urls,
+        final_urls as source_final_urls, 
         replace(replace(final_urls, '[', ''),']','') as final_urls,
+         --Adding final_url_suffix because sometimes the UTMs are only configured in the final_url_suffix than the final_urls
+        final_url_suffix,
```

Comment:
For my data, `final_url_suffix` is always empty. Strangely enough it isn't `null`, it is instead just ` `. Do you see there being a value in adding a case statement here to persist this as a null entry if there is no data included? 

My thoughts are:

```sql
case when final_url_suffix = '' then null end as final_url_suffix
```

Reply:
If you agree with the above, then we should probably make the same update in the campaign_history model

Reply:
We are seeing the same where `final_url_suffix` can be both a null and empty string when not populated. A case or maybe a `nullif(final_url_suffix, '') as final_url_suffix` seems necessary as you said.

Reply:
Thanks for chiming in @dlubawy! This was a big blocker for us not moving forward and appreciate your confirmation and input. Let me try some options and share back once I have a WIP with this change.